docs/shared/features/enhance-tooling.md

```diff
@@ -0,0 +1,102 @@
+# Enhance Tooling
+
+Nx is able to enhance your existing tooling in a non-intrusive way so that, without modifying your tooling set-up, Nx can add:
+
+- [Fully configured caching](/features/cache-task-results)
+- [Task pipelines](/concepts/task-pipeline-configuration)
+- [Task distribution](/ci/features/distribute-task-execution)
+
+This works for any tool for which there exists an [official plugin](/plugin-registry).
```

Comment:
Can we formulate this slightly different. This sounds like a "either use plugins or you don't get caching, task pipelines and task distribution", which is not entirely true.

It is the "automatic enhancing" that only works with plugins. I want it to be softer though and not be a "no-go" for folks that read the line and are like "oh I need plugins, forget it"